AsyncDisplayKit/TextKit/ASTextKitRenderer.mm

```diff
@@ -144,9 +162,18 @@ - (void)_calculateSize
   if (isinf(_constrainedSize.width) == NO && [_attributes.pointSizeScaleFactors count] > 0) {
     _currentScaleFactor = [[self fontSizeAdjuster] scaleFactor];
   }
-  
-  __block NSTextStorage *scaledTextStorage = nil;
+
+  // If we do not scale, do exclusion, or do custom truncation, we should just use TextKit for a fast-path.
```

Comment:
Is it "just use TextKit", or more like "Just use NSAttributedString's built-in sizing?"  Or is this implemented in Foundation or UIKit as a category?